### PR TITLE
add rigetti agave and aspen

### PR DESCRIPTION
**Pull request by lucasberent**
Adds Rigetti architectures Agave and Aspen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,45 +94,59 @@ Architectures that are available per default (either as strings or under `qmap.A
 - `IBMQ_Bogota` (5 qubit, undirected linear chain layout)
 - `IBMQ_Casablanca` (7 qubit, undirected H-shape layout)
 - `IBMQ_Tokyo` (20 qubit, undirected brick-like layout)
+- `Rigetti_Agave` (8 qubit, bidirected ring layout)
+- `Rigetti_Aspen` (16 qubit, bidirected dumbbell layout)
```
Comment (burgholzer) [inline README.md:98]: this should read `undirected` in both cases.

File: README.md [hunk 2]
```diff
@@ -94,45 +94,59 @@ Architectures that are available per default (either as strings or under `qmap.A
...
 
 Whether the heuristic (*default*) or the exact mapper is used can be controlled by passing `method="heuristic"` or `method="exact"` to the `compile` function.
 
 There are several configuration options that can be passed to the `compile` function:
 
-- The heuristic mapper offers the `initial_layout` option, which allows to choose one of the following strategies for choosing an initial layout:
```
Comment (burgholzer) [inline README.md:102]: Is there a reason for the formatting changes in the rest of this file?

File: include/configuration/AvailableArchitecture.hpp [hunk 3]
```diff
@@ -35,6 +37,10 @@ enum class AvailableArchitecture {
             return "IBMQ_Casablanca";
         case AvailableArchitecture::IBMQ_Tokyo:
             return "IBMQ_Tokyo";
+        case AvailableArchitecture::Rigetti_Agave:
+            return "Rigetti_Agave";
+        case AvailableArchitecture::Rigetti_Aspen:
+            return "Rigetti_Aspen";
```
Comment (burgholzer) [inline include/configuration/AvailableArchitecture.hpp:43]: In order to get the CI checks to pass, you have to add a test that uses these architectures.

**Review by burgholzer — CHANGES_REQUESTED** [review]
Overall this LGTM. many thanks 👍🏼 . A couple of small requests can be found below.

File: README.md (hunk 2 above)
Comment (lucasberent) [reply README.md:102]: not intentional, auto format i assume

**Review by burgholzer — APPROVED** [review]
LGTM

**Pull request merged**